pkg/provider/aws.go

```diff
@@ -159,17 +159,20 @@ func getEC2Client(ctx context.Context, region string) (*ec2.Client, error) {
 }
 
 // TODO: during Plural init we should ask the user to choose which AZs they want to use (first 3, random, manual, look at how CAPA does that). There should be a minimum limit of 3.
-func getAvailabilityZones(ctx context.Context, region string) ([]string, error) {
+func getAvailabilityZones(ctx context.Context, region string, cloudFlag bool) ([]string, error) {
```

Comment:
could we just have this default to `first3` entirely?  It's not piped into `plural up` at all so is actually irrelevant at the moment.  Was only for CAPI if i remember

Reply:
done